packages/snaps-sdk/tsup.config.ts

```diff
@@ -0,0 +1,22 @@
+import deepmerge from 'deepmerge';
+import type { Options } from 'tsup';
+
+import packageJson from './package.json';
+
+// `tsup.config.ts` is not under `rootDir`, so we need to use `require` instead.
+// eslint-disable-next-line @typescript-eslint/no-require-imports, @typescript-eslint/no-var-requires
+const { default: baseConfig } = require('../../tsup.config');
+
+delete baseConfig.entry;
```

Comment:
Why do we need this?

Reply:
`deepmerge` merges arrays by default, so then `tsup` would build all TypeScript files. Since we disable splitting here, we only want `index.ts` to be built.